Google.Cloud.EntityFrameworkCore.Spanner.IntegrationTests/QueryTests.cs

```diff
@@ -555,7 +555,7 @@ public async Task CanUseStringConcat()
 
             var calculatedFullName = await db.Singers
                 .Where(s => new long[] { singerId }.Contains(s.SingerId))
-                .Select(s => s.FirstName + " " + s.LastName)
+                .Select(s => string.Concat(s.FirstName, " ", s.LastName))
```

Comment:
Why do we need this change, given that we've got a test that shows we can use the + operator for strings?

Reply:
Oops. Seems like a left-over from the initial development, when I didn't bother to create separate tests for plus and concat. I've added a separate test for the plus operator.

(The other tests use a mock server, these tests are actual integration tests)